CONTRIBUTING.md

```diff
@@ -28,7 +28,15 @@ First, update [commands.yml](temporalcli/commandsgen/commands.yml) following the
 This will expect every non-parent command to have a `run` method, so for new commands developers will have to implement
 `run` on the new command in a separate file before it will compile.
 
+Once a command is updated, the CI will automatically generate new docs
+and create a PR in the Documentation repo with the corresponding updates. To generate these docs locally, run:
+
+    go run ./temporalcli/internal/cmd/gen-docs
+
+This will auto-generate a new set of docs to `temporalcli/docs/`. If a new root command is added, a new file will be automatically generated, like `temporal activity` and `activity.mdx`.
```

Comment:
You could also accept the dir to write to as a CLI arg to this tool, but up to you